docs/general/dql/selects.rst

```diff
@@ -522,6 +532,14 @@ does always return ``NULL`` when comparing ``NULL``.
     +----------+
     SELECT 1 row in set (... sec)
 
+.. CAUTION::
+
+    ``IS NOT NULL`` clauses can slow a query down because CrateDB has to
+    iterate over all rows for the comparison and cannot utilize the index.
```

Comment:
This is also no longer true. We optimized this with 4.2.0 , see https://github.com/crate/crate/commit/f1b5e03a0592e8ba73c7dd0b5823df6f71de218d